### PR TITLE
Show start stop button for job scheduler

### PR DESCRIPTION
**Pull request by satr**

### CODE REVIEW

File: src/components/component/toolbar.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+interface ToolbarState {
+  startRequestStatus: RequestState;
+  startRequestMessage: string;
+  stopRequestStatus: RequestState;
+  stopRequestMessage: string;
+  restartRequestStatus: RequestState;
+  restartRequestMessage: string;
+}
+
+interface ToolbarProps extends ToolbarData, ToolbarAction, ToolbarState {}
```
Comment (Pespiri) [inline src/components/component/toolbar.tsx:56]: export primary interface
```suggestion
export interface ToolbarProps
  extends ToolbarData,
    ToolbarAction,
    ToolbarState {}
```

File: src/components/component/toolbar.tsx [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    stopComponent: (appName, envName, componentName) =>
+      dispatch(
+        componentActions.stop.stopRequest(appName, envName, componentName)
+      ),
+    restartComponent: (appName, envName, componentName) =>
+      dispatch(
+        componentActions.restart.restartRequest(appName, envName, componentName)
+      ),
+  };
+}
```
Comment (Pespiri) [inline src/components/component/toolbar.tsx:199]: add function return type
```suggestion
function mapDispatchToProps(dispatch: Dispatch): ToolbarAction  {
  return {
    startComponent: (appName, envName, componentName) =>
      dispatch(
        componentActions.start.startRequest(appName, envName, componentName)
      ),
    stopComponent: (appName, envName, componentName) =>
      dispatch(
        componentActions.stop.stopRequest(appName, envName, componentName)
      ),
    restartComponent: (appName, envName, componentName) =>
      dispatch(
        componentActions.restart.restartRequest(appName, envName, componentName)
      ),
  };
}
```

File: src/components/page-active-component/active-component-overview.tsx [hunk 3]
```diff
@@ -25,27 +27,99 @@ import {
 } from '../../state/subscriptions/action-creators';
 import { getEnvsUrl } from '../../utils/routing';
 import { routeWithParams } from '../../utils/string';
+import { Dispatch } from 'redux';
```
Comment (Pespiri) [inline src/components/page-active-component/active-component-overview.tsx:30]: sort external includes on top

File: proxy/server.dev-host-macos.conf [hunk 4]
```diff
@@ -7,9 +7,9 @@ server {
     error_page 500 502 503 504 /50x.html;
 
     location /api/ {
-#        proxy_pass https://server-radix-api-dev.dev.radix.equinor.com; #for k8s hosted API
+       proxy_pass https://server-radix-api-qa.dev.radix.equinor.com; #for k8s hosted API
         #for localhost API on MacOS https://docs.docker.com/desktop/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host
-        proxy_pass http://host.docker.internal:3002;
+#        proxy_pass http://host.docker.internal:3002;
```
Comment (Pespiri) [inline proxy/server.dev-host-macos.conf:12]: are these changes intended? if so ignore my comment

File: src/components/component/toolbar.tsx [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+    stopEnabled: PropTypes.bool,
+    startComponent: PropTypes.func.isRequired,
+    stopComponent: PropTypes.func.isRequired,
+    restartComponent: PropTypes.func.isRequired,
+    startRequestStatus: PropTypes.oneOf(Object.values(RequestState)),
+    startRequestMessage: PropTypes.string,
+    stopRequestStatus: PropTypes.oneOf(Object.values(RequestState)),
+    stopRequestMessage: PropTypes.string,
+    restartRequestStatus: PropTypes.oneOf(Object.values(RequestState)),
+    restartRequestMessage: PropTypes.string,
```
Comment (Pespiri) [inline src/components/component/toolbar.tsx:75]: According to the `ToolbarState` interface these props are supposed to be required
```suggestion
    startRequestStatus: PropTypes.oneOf(Object.values(RequestState)).isRequired,
    startRequestMessage: PropTypes.string.isRequired,
    stopRequestStatus: PropTypes.oneOf(Object.values(RequestState)).isRequired,
    stopRequestMessage: PropTypes.string.isRequired,
    restartRequestStatus: PropTypes.oneOf(Object.values(RequestState))
      .isRequired,
    restartRequestMessage: PropTypes.string.isRequired,
```

**Review by Pespiri — CHANGES_REQUESTED** [review]

File: proxy/server.dev-host-macos.conf (hunk 4 above)
Comment (satr) [reply proxy/server.dev-host-macos.conf:12]: This was invalid link

File: src/components/page-active-component/active-component-overview.tsx (hunk 3 above)
Comment (satr) [reply src/components/page-active-component/active-component-overview.tsx:30]: Fixed

**Review by Pespiri — APPROVED** [review]
